### PR TITLE
code_review_test

### PR DESCRIPTION
**Pull request by mattblasa**
Leave comment if you see this. Testing code review system.

### CODE REVIEW

**Comment by curlycuckoo** [comment]
testing.

File: .ipynb_checkpoints/India_House_Data-checkpoint.ipynb [hunk 1]
```diff
@@ -2986,33 +2986,336 @@
...
+       "15    11494.252874  \n",
+       "16     5952.380952  \n",
+       "17     8238.993711  \n",
+       "18     9633.027523  \n",
+       "19     7500.000000  "
+      ]
+     },
+     "execution_count": 78,
+     "metadata": {},
+     "output_type": "execute_result"
```
Comment (curlycuckoo) [inline .ipynb_checkpoints/India_House_Data-checkpoint.ipynb:3284]: testing

File: .ipynb_checkpoints/India_House_Data-checkpoint.ipynb [hunk 2]
```diff
@@ -2986,33 +2986,336 @@
...
    ],
    "source": [
-    "df8 = remove_bhk_outliers(df7)"
+    "df8.head(20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 79,
+   "metadata": {},
```
Comment (curlycuckoo) [inline .ipynb_checkpoints/India_House_Data-checkpoint.ipynb:3294]: testing *4

**Review by curlycuckoo — APPROVED** [review]
testing

File: .ipynb_checkpoints/India_House_Data-checkpoint.ipynb (hunk 1 above)
Comment (mattblasa) [reply .ipynb_checkpoints/India_House_Data-checkpoint.ipynb:3284]: testing

File: .ipynb_checkpoints/India_House_Data-checkpoint.ipynb (hunk 2 above)
Comment (mattblasa) [reply .ipynb_checkpoints/India_House_Data-checkpoint.ipynb:3294]: I can see your comment

**Pull request merged**